clients/windows/src/input/mod.rs

```diff
@@ -5,5 +5,4 @@ pub mod message;
 pub mod mouse;
 pub mod pointer;
 
-pub use message::Message;
```

Comment:
I checked out the parent module and other submodules and didn't see anything taking advantage of this. It gets imported as message::{Message,...

Reply:
So I'm realizing the value of this PR is higher than I expected -- seems like we have a large body of windows code that was never being clippy'd.

So I imagine you came about this line via clippy? If clippy told you to remove this that's totally fine. Does the rest of clippy pass now?

Reply:
Yeah, clippy was telling me to remove it, and clippy now passes after I made all the instructed changes